app/models/config-model.js

```diff
@@ -286,6 +287,8 @@ if ( config[ 'base path' ] && config[ 'base path' ].lastIndexOf( '/' ) === confi
 }
 config[ 'offline path' ] = '/x';
 
+config['root'] = pkgDir.sync( __dirname );
+
```

Comment:
I figured, it was best to make this centrally accessible in the same way as other such variables (though there is probably scope to rethink that whole model - but beyond the scope of this PR ;)).